charts/vineyard/templates/worker/statefulset.yaml

```diff
@@ -150,9 +153,21 @@ spec:
           volumeMounts:
           - name: vineyard-socket-volume
             mountPath: /var/run/vineyard
+          - name: log
+            mountPath: /var/log/vineyard
           {{- if eq (include "vineyard.checkTieredStore" .) "true" }}
 {{- include "vineyard.worker.tieredstoreVolumeMounts" . }}
           {{- end }}
+        - name: metrics
+          image: vineyardcloudnative/vineyardd-grok-exporter
+          imagePullPolicy: {{ .Values.worker.imagePullPolicy }}
+          ports:
+          - name: exporter
+            containerPort: {{ int .Values.worker.ports.exporter }}
+            protocol: TCP
+          volumeMounts:
+          - name: log
```

Comment:
How can we prevent log files from occupying a significant amount of disk space? Is there a log rotation strategy？

Reply:
At present not. I think we can use such [logrotate](https://linux.die.net/man/8/logrotate) command to rotate the log to ensure that its size does not exceed a threshold in the exporter container. What do you think?

Reply:
If we don't have a good solution for metrics, I think we can disable it now. And let's design it well, then put it next release of Fluid.

Reply:
Okay, I have disabled the metrics in the latest commit.

Reply:
In the future, vineyardd should be able to support rotate logs, maybe in the next version.